lib/src/model/nutriments.dart

```diff
@@ -15,8 +17,20 @@ class Nutriments extends JsonObject {
         final String tag = _getTag(nutrient, perSize);
         try {
           final double? value = JsonObject.parseDouble(map[tag]);
+
           if (value != null) {
-            _map[tag] = value;
+            _map[tag] = NutrimentValue._fromMap(map, tag);
+            continue;
```

Comment:
Maybe wrong, not sure, depends on the received data:
1. are the value and the modifier merged together into a single `String`, like `'sugars_100g': '<0.5'`
2. or do we get the value in a field and the modifier in another, like `'sugars_100g': 0.5,'sugars_100g_modifier':'<'`

If it's (2), the `continue` is wrong.